sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -176,6 +176,48 @@
     "will not create a docker container using the image defined in the environment."
 )
 
+WORKSPACE_EXCLUSIVE_OPERATIONS = {
+    "_batch_deployment_operations",
+    "_batch_endpoint_operations",
+    "_compute_operations",
+    "_datastore_operations",
+    "_job_operations",
+    "_model_dataplane_operations",
+    "_online_deployment_operations",
+    "_online_endpoint_operations",
+    "_run_operations",
+    "_schedule_operations",
+}
+WORKSPACE_OR_REGISTRY_REQUIRED_OPERATIONS = {
+    "_code_operations",
+    "_component_operations",
+    "_data_operations",
+    "_environment_operations",
+    "_model_operations",
+}
+WORKSPACE_NOT_INCLUDED_ERROR_MESSAGE = (
+    "\n\nThis operation requires that you specify a connection to a workspace. "
```

Comment:
Nit-pick: The word of "connection" is abused in the industry ;) Let's remove "a connection" from the message.